### DISCUSSION TITLE
Missing OTUs

### DISCUSSION THREAD

**Discussion by avi-dhar** [FrederickHuangLin/ANCOMBC#137] [General]
Hi All,
Wishing you all a very happy new year !!!

I had a query about my ANCOMBC analysis. 
I have around 576 OTUs and after running ANCOMBC on my data, results for just 476 come up. There are around 100 missing OTUs. Is this normal ? In case if it is, what is the criteria behind omitting OTUs. My command for ANCOMBC is - 

out = ancombc(data = carbom, assay_name = "counts", tax_level = NULL, formula = "Treatment", group = "Treatment" )

I have 4 treatments - a. Control b. Drought c. Warming and d. Warming/Drought.

Thanks 
Avinash

**Comment by FrederickHuangLin** [comment]
Hi Avinash,

`ancombc` will filter taxa by their prevalences. By default `prv_cut = 0.1`, which means taxa with prevalences less than 10%
will be excluded from the analysis. 

Best,
Huang
